src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -81,86 +92,169 @@ master_create_distributed_table(PG_FUNCTION_ARGS)
 	text *distributionColumnText = PG_GETARG_TEXT_P(1);
 	Oid distributionMethodOid = PG_GETARG_OID(2);
 
```

Comment:
Maybe a single line comment that `master_create_distributed_table()` cannot create colocated tables that's why we always pass `INVALID_COLOCATION_ID `?